src/cdef_tmpl_16.rs

```diff
@@ -248,13 +248,14 @@ pub struct Dav1dCdefDSPContext {
     pub dir: cdef_dir_fn,
     pub fb: [cdef_fn; 3],
 }
-pub const DAV1D_X86_CPU_FLAG_AVX512ICL: CpuFlags = 16;
-pub const DAV1D_X86_CPU_FLAG_SSE2: CpuFlags = 1;
-pub const DAV1D_X86_CPU_FLAG_AVX2: CpuFlags = 8;
-pub const DAV1D_X86_CPU_FLAG_SSE41: CpuFlags = 4;
-pub const DAV1D_X86_CPU_FLAG_SSSE3: CpuFlags = 2;
-pub type CpuFlags = libc::c_uint;
-pub const DAV1D_X86_CPU_FLAG_SLOW_GATHER: CpuFlags = 32;
+cfg_if! {
```

Comment:
I would prefer if you could delete all of this and instead add `use crate::src::x86::cpu::*;` at the beginning of `cdef_dsp_init_x86`. That would not only be less code, it would also ensure that the `cfg_if` parts match. Right now, you define these constants if we're on x86 or x86-64 but `cdef_dsp_init_x86` is only defined under a more narrow set of circumstances.  Same fix should be applied in the corresponding 8-bit file.

Reply:
I moved all of the `cfg_if!`ed `use`s to inside the `*_init_x86` functions now, for all of them in all of the files.  I didn't realize they were only used in these functions, so I agree it's better to put them inside so the `cfg`s match exactly.  I didn't use glob imports, though, as since this isn't a real Rust `enum`, we'd have to import the whole `mod src::x86::cpu`, which contains various other items, too.